config/jobs-cip.yaml

```diff
@@ -21,9 +21,162 @@ _anchors:
       cross_compile: 'arm-linux-gnueabihf-'
       defconfig: multi_v7_defconfig
 
+  kbuild-gcc-12-arm64-job: &kbuild-gcc-12-arm64-job
+    <<: *kbuild-job
+    image: ghcr.io/kernelci/{image_prefix}gcc-12:arm64-kselftest-kernelci
+    params: &kbuild-gcc-12-arm64-params
+      arch: arm64
+      compiler: gcc-12
+      cross_compile: 'aarch64-linux-gnu-'
+      defconfig: defconfig
+      fragments:
+        - 'lab-setup'
+        - 'kselftest'
+
+  kbuild-gcc-12-x86-job: &kbuild-gcc-12-x86-job
+    <<: *kbuild-job
+    image: ghcr.io/kernelci/{image_prefix}gcc-12:x86-kselftest-kernelci
+    params: &kbuild-gcc-12-x86-params
+      arch: x86_64
+      compiler: gcc-12
+      defconfig: x86_64_defconfig
+      fragments:
+        - 'kselftest'
+
 jobs:
   baseline-x86-cip: *baseline-job
 
+# start of CIP x86 configuration
+  kbuild-gcc-12-x86-allnoconfig: &kbuild-gcc-12-x86-allnoconfig-job
+    <<: *kbuild-gcc-12-x86-job
+    params:
+      <<: *kbuild-gcc-12-x86-params
+      defconfig: allnoconfig
+    rules:
+      tree:
+      - 'kernelci'
+      - 'stable-rc'
+      - 'stable'
+
+  kbuild-gcc-12-x86-cip-allnoconfig:
+    <<: *kbuild-gcc-12-x86-allnoconfig-job
+    params:
+      <<: *kbuild-gcc-12-x86-params
+      defconfig:
+        - multi_v7_defconfig
```

Comment:
You've got `multi_v7_defconfig` used in a few places, but isn't this just an Arm32 config?